### PR TITLE
[TGBOT-276] Create parent pom.xml

### PR DESCRIPTION
**Pull request by Sav4ykVladzislau**
https://schneidervs.atlassian.net/browse/TGBOT-276

### CODE REVIEW

**Review by yurasava — DISMISSED** [review]

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.springframework</groupId>
+    <artifactId>stoservice-multi-module</artifactId>
+    <version>0.1.0</version>
```
Comment (max-java) [inline pom.xml:8]: keep "snapshot" in version

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    <packaging>pom</packaging>
+    <name>stoservice-multimodule</name>
+
+    <modules>
+<!--        Added you module to this block-->
+
+    </modules>
+
+</project>
```
Comment (max-java) [inline pom.xml:17]: add new line at the end of the file

**Comment by max-java** [comment]
minor comments, otherwise lgfm

**Review by max-java — APPROVED** [review]

**Review by yurasava — DISMISSED** [review]

**Review by max-java — APPROVED** [review]
lgfm

**Review by schneidervs — APPROVED** [review]
Maybe

**Review by yurasava — APPROVED** [review]

**Pull request merged**